src/common/buffer.cc

```diff
@@ -1118,6 +1118,12 @@ static simple_spinlock_t buffer_debug_lock = SIMPLE_SPINLOCK_INITIALIZER;
     return true;
   }
 
+  bool buffer::list::can_zero_copy(const char *dst) const
```

Comment:
@jdurgin. Yes,  it's checking whether the bufferlist only uses the provided buffer. So i use can_zero_copy . But the name is_buf i think not good.